tests/v2_validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -748,12 +748,14 @@ def validate_remove_service_link(admin_client, service, consumedService):
     service_maps = admin_client. \
         list_serviceConsumeMap(serviceId=service.id,
                                consumedServiceId=consumedService.id)
-    assert len(service_maps) == 1
+    assert len(service_maps) == 0
+    """
```

Comment:
list does not return any removed entries anymore. Earlier we were getting the serviceConsumeMap entry and making sure its state was removed .I have changed this to make sure that there is no entry returned for this instead.